src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/contentcard/ContentCard.kt

```diff
@@ -38,8 +39,9 @@ class ContentCard(initialContent: Content? = null) : VBox() {
     }
 }
 
-fun contentcard(verse: Content, init: ContentCard.() -> Unit): ContentCard {
+fun Pane.contentcard(verse: Content, init: ContentCard.() -> Unit): ContentCard {
```

Comment:
see above

Reply:
I have no reasoning behind it. I was just following an implementation that Matthew had adopted previously